### PR TITLE
I got our node server connected to my Mongodb

### PR DESCRIPTION
**Pull request by julianfortune**
- Uses my mongodb for this class
- I think we should not put the password in the repo since this repo is public. I will send password through GroupMe
- To get the project building, you need to:

1.  Connect to the OSU vpn/be on campus
2. Put a file called `prod-password.sh` in the environment folder with 775 permissions with the text `export MONGODB_PASS='<here>'`. This file will be ignored by Git so you can't accidentally commit it. 
3. Run `. ./environment/production.sh`. The `.` at the beginning specifies the script should be run in the current shell, and otherwise the variables don't show up. I don't know how to integrate this with the npm build shortcut, since that seems to run in a sub shell.

### CODE REVIEW

File: environment/prod.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# 2. Run this script using `. ./environment/production.sh`
+#    Without the leading `.` the variables will be set in a sub shell,
+#    not your currently running shell.
+# 3. Start the server `npm start` `node server.js` or `nodemon server.js`
+
+# Set env variables
+export MONGODB_HOST='classmongo.engr.oregonstate.edu'
+export MONGODB_NAME='cs290_fortunej'
+export MONGODB_USER='cs290_fortunej'
+. ./environment/prod-password.sh
```
Comment (julianfortune) [inline environment/prod.sh:12]: This is that file you need to create

File: server.js [hunk 2]
```diff
@@ -49,6 +64,18 @@ app.get('*', function(req, res) {
...
-});
+// Connect to the database and set the gloabl var so it can be used be the whole
+// middleware stack.
+MongoClient.connect(mongoDBURL, function(err, client) {
+    if (err) {
+        throw err
+    }
+
+    database = client.db(mongoDBName)
+    console.log("☁️  Connected to database.")
```
Comment (julianfortune) [inline server.js:75]: I copied most of the mongodb in node code from Rob

File: environment/prod.sh (hunk 1 above)
Comment (merezal) [reply environment/prod.sh:12]: Easy enough, so we have to run this file manually for each session?

**Review by merezal — APPROVED** [review]
Looks good

File: environment/prod.sh (hunk 1 above)
Comment (julianfortune) [reply environment/prod.sh:12]: Unfortunately yes. I can't think of how to set env variables without manually running this. The problem is scripts that run run in different shells without the `.` prefix, and so the env variables get set there, and then that sub-shell gets killed.

**Pull request merged**